zio-http/src/main/scala/zio/http/ServerConfig.scala

```diff
@@ -131,4 +146,46 @@ object ServerConfig {
     case object PARANOID extends LeakDetectionLevel
   }
 
+  final case class ResponseCompressionConfig(
+    contentThreshold: Int = 0,
+    options: IndexedSeq[CompressionOptions] = IndexedSeq.empty,
+  )
+
+  final case class CompressionOptions(
```

Comment:
Would be nice to have a scaladoc comment about the valid range / meaning of these parameters (also for `contentThreshold` above)

Reply:
Thanks, @vigoo.